packages/react/src/components/IonToast.tsx

```diff
@@ -20,15 +20,7 @@ export interface ToastOptions extends Omit<ToastOptionsCore, 'buttons'> {
   buttons?: (ToastButton | string)[];
 }
 
-const toastController = {
-  create: (options: ToastOptions) => toastControllerCore.create(options as any),
-  dismiss: (data?: any, role?: string | undefined, id?: string | undefined) =>
-    toastControllerCore.dismiss(data, role, id),
-  getTop: () => toastControllerCore.getTop(),
-};
-
-export const IonToast = /*@__PURE__*/ createControllerComponent<ToastOptions, HTMLIonToastElement>(
+export const IonToast = /*@__PURE__*/ createInlineOverlayComponent<JSX.IonActionSheet, HTMLIonActionSheetElement>(
```

Comment:
The React build is likely failing from this copy & paste. Should reference the Toast component and HTML element type.

Reply:
Ah, good catch 👀